docs/enterprise/reference/config.md

```diff
@@ -17,6 +17,12 @@ a list of user ids, names or emails to make administrators, useful for bootstrap
 
 **Default value:** `none`
 
+## audience
+
+audience for verifying signing key
+
+**Default value:** `[]`
```

Comment:
we should probably stress this can be a list.

Reply:
https://github.com/pomerium/pomerium/pull/2599/commits/b57d9d57549c136080528e99b024e512651b4b19